SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -1634,8 +1634,11 @@ \section{Initialization and Assignment}
 \code{\#\#nameOn}/\code{\#\#nameOff} method is recommended instead of directly
 manipulating the data member.
 
-Smart pointers need not to be initialized, since they initialize themselves
-to the \code{null} pointer, so they are the sole exception to the above rule.
+An exception is also made for low level utility classes for which data member initialization would
+cause a significant performance penalty. This is why for example \code{FixedArray::m_InternalArray}
+and \code{Index::m_InternalArray} do not have a default member initializer. But when a class has a
+static `New()` member function to instantiate objects (as defined by \code{itkNewMacro}), it should
```

Comment:
One thing to note: LaTeX requires two backticks+two upright ticks to enclose a word within double ticks: ``''.

Reply:
Addressed in 992149c.